package.json

```diff
@@ -3,7 +3,7 @@
   "license": "MIT",
   "scripts": {
     "postinstall": "lerna bootstrap",
-    "ts:lint": "tslint \"packages/**/*.tsx\" \"packages/**/*.ts\"",
+    "ts:lint": "tslint 'packages/**/*.tsx' 'packages/**/*.ts' --exclude 'packages/**/definitions/**/*'",
```

Comment:
pls use double quotes - single quotes doesn't work on windows 

In my branch I have something like this:
```
"ts:lint": "tslint  --project tsconfig.json --type-check --exclude \"packages/*/node_modules/**\" \"packages/**/*.tsx\" \"packages/**/*.ts\""
```